pipeline/inferrer/aspect_ratio_inferrer/README.md

```diff
@@ -0,0 +1,14 @@
+# Infer aspect ratio
+
+```
+docker build -t aspect_ratio_api .
+docker run -p 80:80 aspect_ratio_api
+```
+
+```
+curl "http://0.0.0.0/aspect_ratio/?image_url=SOME_ENCODED_IMAGE_URL"
+```
+
+## TODO
+
+Could avoid fetching the full image by using the info.json when given a IIIF URL.
```

Comment:
Another solution might be to get a smaller version of the image - although I am not sure where we might loose accuracy. Or maybe read it from the metadata of the smaller image... not sure it's an issue, but just some thoughts.

Reply:
Yeah, I think a combination of smaller-image-fetching and IIIF would do a lot to speed this up without meaningful decrease in accuracy. The current state feels like a reasonable MVP for now though.

Reply:
Remember that we're fetching the large-ish images anyway and the URLs will, in reality, be filesystem paths. Obviously HTTP requests are waaaayyyy slower than file IO - so I don't think fetching a smaller image or a JSON file would be faster in anger, this is probably the most performant at the moment.